cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/NewCommandTest.java

```diff
@@ -721,6 +721,31 @@ public void testNewCommandWithInvalidPackageNames1(String packageName, String ou
         Assert.assertEquals(buildLog.replaceAll("\r", ""), getOutput(outputLog));
     }
 
+    @DataProvider(name = "PackageNameHasOnlyAlphanumeric")
+    public Object[][] providePackageNameHasOnlyAlphanumeric() {
+        return new Object[][] {
+                { "#", "my_package" },
+                { "_", "my_package" }
+        };
+    }
+    @Test(description = "Test init command with package name has alpha-numeric characters",
+            dataProvider = "PackageNameHasOnlyAlphanumeric")
+    public void testInitCommandWithPackageNameHasOnlyAlphanumeric(String pkgName, String derivedPkgName)
```

Comment:
```suggestion
    public void testInitCommandWithPackageNameHasOnlyNonAlphanumeric(String pkgName, String derivedPkgName)
```

Reply:
Applies to other places having a similar pattern

Reply:
Applied